step.yml

```diff
@@ -181,14 +181,25 @@ inputs:
         - "up_until_maximum_repetitions"
   - maximum_test_repetitions: 3
     opts:
-      title: Maximum Test Repetitions
+      title: Maximum Test Repetitions (Available in Xcode 13+)
```

Comment:
same as above

Reply:
Let's continue the discussion in that thread about this topic!